spec/system/department_admin_should_be_able_to_manage_assemblies_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Admin manages assemblies', versioning: true, type: :system do
+  let(:organization) { create(:organization) }
+  let(:area) { create(:area, organization: organization) }
+  let(:department_admin) { create(:department_admin, :confirmed, organization: organization, area: area) }
+
+  before do
+    switch_to_host(organization.host)
+    login_as department_admin, scope: :user
+  end
+
+  shared_examples 'creating an assembly' do
+    let(:image1_filename) { 'city.jpeg' }
+    let(:image1_path) { Decidim::Dev.asset(image1_filename) }
+
+    let(:image2_filename) { 'city2.jpeg' }
+    let(:image2_path) { Decidim::Dev.asset(image2_filename) }
+
+    before do
+      visit decidim_admin_assemblies.assemblies_path
+      click_link 'New assembly'
+    end
+
+    it 'creates a new assembly' do
+      within '.new_assembly' do
+        fill_in_i18n(
+          :assembly_title,
+          '#assembly-title-tabs',
+          en: 'My assembly',
+          es: 'Mi proceso participativo',
+          ca: 'El meu procés participatiu'
+        )
+        fill_in_i18n(
+          :assembly_subtitle,
+          '#assembly-subtitle-tabs',
+          en: 'Subtitle',
+          es: 'Subtítulo',
+          ca: 'Subtítol'
+        )
+        fill_in_i18n_editor(
+          :assembly_short_description,
+          '#assembly-short_description-tabs',
+          en: 'Short description',
+          es: 'Descripción corta',
+          ca: 'Descripció curta'
+        )
+        fill_in_i18n_editor(
+          :assembly_description,
+          '#assembly-description-tabs',
+          en: 'A longer description',
+          es: 'Descripción más larga',
+          ca: 'Descripció més llarga'
+        )
+
+        fill_in :assembly_slug, with: 'slug'
+        fill_in :assembly_hashtag, with: '#hashtag'
+        attach_file :assembly_hero_image, image1_path
+        attach_file :assembly_banner_image, image2_path
+
+        find('*[type=submit]').click
+      end
+
+      expect(page).to have_admin_callout('successfully')
+      expect(Decidim::Assembly.last.area).to eq(area)
+
+      within '.container' do
+        # expect(page).to have_current_path decidim_admin_assemblies.assemblies_path(parent_id: parent_assembly&.id)
```

Comment:
remove

Reply:
I had some problems with this path validation in the test environment. I've checked in development and perprod and it works. Maybe we can leave it here just in case we have more time in the future to make it work?